editor/filesystem_dock.cpp

```diff
@@ -1064,6 +1064,13 @@ void FileSystemDock::_folder_option(int p_option) {
 				child = child->get_next();
 			}
 			break;
+		case FOLDER_SHOW_IN_EXPLORER:
+			String path = item->get_metadata(tree->get_selected_column());
+			String dir = GlobalConfig::get_singleton()->globalize_path(path);
+			dir = dir.substr(0, dir.find_last("/"));
```

Comment:
I was not certain if I shoulded do this, but yeah it makes more sense if it would be more "**open in file browser**. So ,I`m going to remove that line.